### PR TITLE
Create product card

### PR DESCRIPTION
**Pull request by ivan-kadykalo**
- [DEMO LINK](https://ivan-kadykalo.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://ivan-kadykalo.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,64 @@
...
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="html">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <!-- Fonts -->
```
Comment (YegorKochetkov) [inline src/index.html:8]: redundant comments (also below the document)

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,64 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <!-- Style -->
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <!-- Body -->
+  <body class="body">
+    <form
```
Comment (YegorKochetkov) [inline src/index.html:17]: in fact, it is not a form. better use section or article

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,64 @@
...
+      action="#card"
+      method="post"
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Product card image"
+        class="image card__image"
```
Comment (YegorKochetkov) [inline src/index.html:26]: redundant class `image`

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,64 @@
...
+      >
+      <h2 class="name card__title">
```
Comment (YegorKochetkov) [inline src/index.html:28]: redundant class `name`
(also below the document)

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,64 @@
...
+          <div class="score__star"></div>
+          <div class="score__star"></div>
+          <div class="score__star"></div>
+        </div>
+        <span class="score__feedback">
+          Отзывов: 5
+        </span>
+      </div>
+
+      <div class="price card__price">
```
Comment (YegorKochetkov) [inline src/index.html:47]: redundant class. you have `card__price`, why do you need `price`?

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,64 @@
...
+        <span class="price__name">
+          Цена:
+        </span>
+        <h3 class="price__value">
+          69 999 грн
+        </h3>
+      </div>
+      <a
+        href="#"
+        class="button card__button"
```
Comment (YegorKochetkov) [inline src/index.html:57]: redundant class

File: src/style_blocks/elements.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  padding: 12px;
+  height: 40px;
```
Comment (YegorKochetkov) [inline src/style_blocks/elements.css:63]: better put first block model modifier, then font modifier

**Review by YegorKochetkov — CHANGES_REQUESTED** [review]
good job)

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,61 @@
...
+    >
+      <img
+        src="images/imac.jpeg"
+        alt="Product card image"
+        class="card__image"
+      >
+      <h2 class="card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+      <p class="code card__code">
```
Comment (MaksimusCRS) [inline src/index.html:28]:
```suggestion
      >
      
      <h2 class="card__title">
        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </h2>
      
      <p class="code card__code">
```

Comment (MaksimusCRS) [reply src/index.html:28]: Делай отступы между тегами, что бы было легче читать твой код

**Review by MaksimusCRS — APPROVED** [review]
Good job!

**Pull request closed without merging**